.github/workflows/cloud-runner-async-checks.yml

```diff
@@ -0,0 +1,58 @@
+name: Async Checks API
+
+on:
+  workflow_dispatch:
+    inputs:
+      checksObject:
+        description: ''
+        required: false
+        default: ''
+
+permissions: write-all
```

Comment:
Is this granular enough? I don't think it's a good idea to give it admin access to the repo, just the checks api

Reply:
true, the main workflow also currently has this for development only. 

Checks will do here.

Needs workflows + dispatch in the main repo.